cmd/oras/tag/tag.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tag
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"oras.land/oras/cmd/oras/internal/errors"
+	"oras.land/oras/cmd/oras/internal/option"
+)
+
+type tagOptions struct {
+	option.Common
+	option.Remote
+
+	targetRef string
+	tagRef    string
```

Comment:
nit: The naming looks a bit confusing to me as I can't tell which is the original ref and which is the new one. Would something like `srcRef`, `dstRef` be clearer?